experiments/bandpass/bandpass_by_task.m

```diff
@@ -37,12 +42,69 @@ function bandpass_by_task(n_steps, window_steps, args_path, task_info, extra_opt
     % Store mean performance
     means(:, si) = cellfun(@(x) mean(x), tst.stats.rej.p.err.vals);
     stderrs(:, si) = cellfun(@(x) std(x), tst.stats.rej.p.err.vals) / sqrt(tst.mSets.runs);
-end
+    
+    % Do breakdown for task type
+    mSets = tst.mSets;
+    TLAB = mSets.data.test.TLAB;
+    nSig = size(tst.models, 2);
+    ps = [tst.models.p];
+    o = [ps.output]; %this combines all sigmas together... need a smarter fix for multiple sigmas
+    temp   = de_calcPErr( vertcat(o.test), mSets.data.test.T, 2);
+    
+    if (strcmp(tst.mSets.expt, 'slotnick_etal_2001'))
```

Comment:
If it's not one of these experiments, this code will not define `trial_types` and therefore fail, no?